From e6bdcd4f67d2fa9812baacdcc3148486aae15d1c Mon Sep 17 00:00:00 2001
From: Honore Tricot <honorex.tricot@intel.com>
Date: Thu, 30 Aug 2012 17:15:28 +0200
Subject: [PATCH 21/78] Revert "Revert "WIFI: Move calibration file to
 /factory""

BZ: 54678

Partition /factory is now available, so re-activate wifi calibration
to factory in order to avoid issue when phone is encrypted.

This reverts commit af24de2e99d7b62658242c254fde71ad3e618f9b.

Change-Id: I6dd39e52211b1ac84c7aba6bb1dca29b600fd00a
Signed-off-by: Honore Tricot <honorex.tricot@intel.com>
Reviewed-on: http://android.intel.com:8080/64069
Reviewed-by: Lemetayer, Jean-marieX <jean-mariex.lemetayer@intel.com>
Reviewed-by: Garcia, JeremieX <jeremiex.garcia@intel.com>
Reviewed-by: Trivelly, Jean <jean.trivelly@intel.com>
Reviewed-by: Bachot, Jean-michel <jean-michel.bachot@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/77978
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 wlan_prov/Android.mk          |    2 +-
 wlan_prov/wlan_provisioning.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/wlan_prov/Android.mk b/wlan_prov/Android.mk
index ed94446..3b1e09a 100644
--- a/wlan_prov/Android.mk
+++ b/wlan_prov/Android.mk
@@ -36,7 +36,7 @@ $(SYMLINKS): $(LOCAL_INSTALLED_MODULE) $(LOCAL_PATH)/Android.mk
 	@echo "Symlink: $@"
 	@mkdir -p $(dir $@)
 	@rm -rf $@
-	$(hide) ln -sf $(addprefix /data/misc/firmware/ti-connectivity/,$(NVS_FILE)) $@
+	$(hide) ln -sf $(addprefix /factory/wifi/,$(NVS_FILE)) $@
 
 ALL_DEFAULT_INSTALLED_MODULES += $(SYMLINKS)
 ALL_MODULES.$(LOCAL_MODULE).INSTALLED := \
diff --git a/wlan_prov/wlan_provisioning.c b/wlan_prov/wlan_provisioning.c
index b0ef079..a79513b 100644
--- a/wlan_prov/wlan_provisioning.c
+++ b/wlan_prov/wlan_provisioning.c
@@ -37,7 +37,7 @@
 const unsigned char NullMacAddr[MAC_ADDRESS_LEN] = { 0, 0, 0, 0, 0, 0 };
 
 #define WIFI_PATH "/data/misc/wifi"
-const char NVS_file_name[] = "/data/misc/firmware/ti-connectivity/wl1271-nvs.bin";
+const char NVS_file_name[] = "/factory/wifi/wl1271-nvs.bin";
 const char Default_NVS_file_name[] = "/system/etc/wifi/wl1271-nvs.bin";
 const char WLAN_SDIO_BUS_PATH[] = "/sys/bus/sdio/drivers/wl1271_sdio/";
 #define WLAN_DRV_SDIO_NAME "wl1271_sdio"
-- 
1.7.9

